mitodl/mitxonline
# bump webpack-bundle-tracker to 0.4.3 to fix deep-extend alert

#### Pre-Flight checklist


#### What are the relevant tickets?
#229 

#### What's this PR do?
- Fixes the alert regarding `deep-extend` package to be upgraded to at least `0.5.1`
- In order to fix the above alert, this PR upgrades `webpack-bundle-tracker` to the max possible minor version available.

**NOTE(For reviewer):**
Although there is a major version of the `webpack-bundle-tracker=1.4.0` available I just upgraded to a max minor version(0.4.3) which also fixed the security alert because the major version might introduce some breaking changes and ATM we might need that since the alert is already getting resolved with `0.4.3`

#### How should this be manually tested?
- Check that `webpack-stats.json` file gets generated as before
- Check that the webpack bundle compilation is successful
